### ISSUE TITLE
Document native requirements

### ISSUE THREAD

**Issue by drphrozen** [snowplow/dbt-snowplow-unified#48]
## Describe the bug

Please add requirements for native applications

## Steps to reproduce

Tried setting up unified with on our dataset, but we were not using mobile_context 1-0-3.

## Expected results

Since no requirements were given we expected it to just work.

## Actual results

Complained about missing fields introduced in mobile-context 1-0-3

## Screenshots and log output

Truncated dbt output:
```
Runtime Error in model snowplow_unified_base_events_this_run (models/base/scratch/snowplow_unified_base_events_this_run.sql)
  [FIELD_NOT_FOUND] No such struct field `resolution` in `android_idfa`, `app_available_memory`, `apple_idfa`, `apple_idfv`, `available_storage`, `battery_level`, `battery_state`, `carrier`, `device_manufacturer`, `device_model`, `low_power_mode`, `network_technology`, `network_type`, `open_idfa`, `os_type`, `os_version`, `physical_memory`, `system_available_memory`, `total_storage`, `is_portrait`. SQLSTATE: 42704
```

## System information
**The contents of your `packages.yml` file:**
Using 0.4 of unified

**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [ ] bigquery
- [ ] snowflake
- [x] databricks
- [ ] other (specify: ____________)

**The output of `dbt --version`:**
N/A

**The operating system you're using:**
N/A

**The output of `python --version`:**
N/A

## Additional context

This could probably be solved by adding the requirement of having mobile context at 1-0-3, which was introduced in:
android/ios: 5.1.0
react native: 2.0.0

as far as i could deduce

## Are you interested in contributing towards the fix?

Not currently

**Comment by rlh1994** [comment]
Hi @drphrozen , thanks for pointing this out. We're just discussing internally at the moment how best to resolve this, either just via a bump in the requirements or to also support older versions of this entity. We'll update this issue soon.

**Comment by rlh1994** [comment]
@drphrozen We've just released a new version (0.4.2) that should fix this issue and no longer error when using an older version of the schema, please let us know if you see any further issues and thanks for reporting this!
